### PR TITLE
Add RFCs for T-compiler

### PR DESCRIPTION
**Pull request by apiraino**
Adding RFCs for T-compiler

@spastorino

### CODE REVIEW

File: src/agenda.rs [hunk 1]
```diff
@@ -460,7 +460,7 @@ pub fn prioritization<'a>() -> Box<dyn Action> {
     });
 
     queries.push(QueryMap {
-        name: "i_nominated_t_compiler",
+        name: "nominated_t_compiler",
```
Comment (apiraino) [inline src/agenda.rs:463]: unrelated, I've renamed all `i-nominated_*` to `nominated_*` to have all the same label

File: templates/prioritization_agenda.tt [hunk 2]
```diff
@@ -104,7 +104,10 @@ tags: weekly, rustc
...
 [T-compiler](https://github.com/rust-lang/rust/issues?q=is%3Aopen+label%3AI-nominated+label%3AT-compiler)
-{{-issues::render(issues=i_nominated_t_compiler, empty="No nominated issues for `T-compiler` this time.")}}
+{{-issues::render(issues=nominated_t_compiler, empty="No nominated issues for `T-compiler` this time.")}}
 
 [libs-impl](https://github.com/rust-lang/rust/issues?q=is%3Aopen+label%3AI-nominated+label%3Alibs-impl)
-{{-issues::render(issues=i_nominated_libs_impl, empty="No nominated issues for `libs-impl` this time.")}}
+{{-issues::render(issues=nominated_libs_impl, empty="No nominated issues for `libs-impl` this time.")}}
+
+[RFC](https://github.com/rust-lang/rfcs/issues?q=is%3Aopen+label%3AI-nominated+label%3AT-compiler)
+{{-issues::render(issues=nominated_rfcs_t_compiler, empty="No nominated RFCs for `T-compiler` this time.")}}
```
Comment (apiraino) [inline templates/prioritization_agenda.tt:113]: here will appear the RFCs for compiler team

**Comment by apiraino** [comment]
@spastorino feel free to try it

rendered:

File: src/agenda.rs [hunk 3]
```diff
@@ -525,7 +536,7 @@ pub fn lang<'a>() -> Box<dyn Action> {
 
     //https://github.com/rust-lang/rfcs/pulls?q=is%3Aopen+is%3Apr+label%3AI-nominated+label%3AT-lang
     queries.push(QueryMap {
-        name: "nominated_rfcs",
+        name: "nominated_rfcs_t_lang",
```
Comment (spastorino) [inline src/agenda.rs:539]: I wouldn't change this one because it doesn't belong to prioritization wg.

File: templates/lang_agenda.tt [hunk 4]
```diff
@@ -24,7 +24,7 @@ Review the [Lang team project board](https://github.com/rust-lang/lang-team/proj
 
 ## Nominated RFCs
 
-{{-issues::render(issues=nominated_rfcs, indent="  ", empty="No nominated RFCs this time.")}}
+{{-issues::render(issues=nominated_rfcs_t_lang, indent="  ", empty="No nominated RFCs this time.")}}
```
Comment (spastorino) [inline templates/lang_agenda.tt:27]: Wouldn't change this file either

**Review by spastorino — COMMENTED** [review]
Other than my minor comments looks great :).

File: src/agenda.rs (hunk 3 above)
Comment (apiraino) [reply src/agenda.rs]: I thought it was more intuitive to have these labels with similar names but I won't bikeshed names, so this change will be reverted

**Comment by apiraino** [comment]
@spastorino applied your suggestion

**Review by spastorino — APPROVED** [review]

**Pull request merged**